doc/manual/preprocessing/ssp.rst

```diff
@@ -126,3 +115,77 @@ decomposition, and employ the eigenvectors corresponding to the
+
+.. math::	{b}^{j}_{s}(t) = b^{j}(t) - \frac{1}{n}\sum_{k}{b^k(t)}
+   :label: eeg_proj
+
+where the noise term :math:`b_{n}^{j}(t)` is given by
+
+.. math:: 	b_{n}^{j}(t) = \frac{1}{n}\sum_{k}{b^k(t)}
+   :label: noise_term
+
+Thus, the projector vector :math:`P_{\perp}` will be given by :math:`P_{\perp}=\frac{1}{n}[1, 1, ..., 1]`
+
+The API
+=======
+
+Once a projector is applied on the data, it is said to be `active`.
+
+The proj attribute
+------------------
+
+It is available in all the basic data containers: ``Raw``, ``Epochs`` and ``Evoked``. It is ``True`` if at least one projector is present and all of them are `active`. 
+
+Computing projectors
+--------------------
+
+In MNE-Python SSP vectors can be computed using general
+purpose functions :func:`mne.compute_proj_epochs`,
+:func:`mne.compute_proj_evoked`, and :func:`mne.compute_proj_raw`.
+The general assumption these functions make is that the data passed contains
+raw, epochs or averages of the artifact. Typically this involves continues raw
+data of empty room recordings or averaged ECG or EOG artefacts.
+
+A second set of highlevel convenience functions is provided to compute projection vector for typical usecases. This includes :func:`mne.preprocessing.compute_proj_ecg` and :func:`mne.preprocessing.compute_proj_eog` for computing the ECG and EOG related artifact components, respectively. For computing the eeg reference signal, the function :func:`mne.proj.make_eeg_average_ref_proj` can be used. The underlying implementation can be found in :mod:`mne.preprocessing.ssp`.
+
+Adding/removing projectors
+--------------------------
+
+To explicitly add a ``proj``, use ``add_proj``. For example::
+
+    >>> projs = mne.read_proj('proj_a.fif')
+    >>> evoked.add_proj(projs)
+
+If projectors are already present in the raw `fif` file, it will be added to the ``info`` dictionary automatically. To remove existing projectors, you can do::
+
+	>>> evoked.add_proj([], remove_existing=True)
+
+Applying projectors
+-------------------
+
+Projectors can be applied at any stage of the pipeline. When the ``raw`` data is read in, the projectors are not applied by default but this flag can be turned on. However, at the ``epochs`` stage, the projectors are applied by default.
+
+To apply explicitly projs at any stage of the pipeline, use ``apply_proj``. For example::
+
+	>>> evoked.apply_proj()
+
+The projectors might not be applied if data are not :ref:`preloaded <memory>`. In this case, it's the ``_projector`` attribute that indicates if a projector will be applied when the data is loaded in memory. If the data is already in memory, then the projectors applied to it are the ones marked as `active`. As soon as you've applied the projectors, it will stay active in the remaining pipeline.
+
+.. Warning:: Once a projection operator is applied, it cannot be reversed.
```

Comment:
another pitfall to mention is that all projections present in a file gets applied during inverse computation. No matter if they are active or not. So if you don't want a certain projection, remove it from the info !